src/locales/en.json5

```diff
@@ -436,7 +436,9 @@
     // {version} is a version number.
     "versionShort": "v{version}",
     // This text is shown on its own before details about a warning.
-    "warning": "Warning"
+    "warning": "Warning",
+    // Part of the error message format that lists the duplicate property name like "<PropertyName> (existing: <ExistingPropertyName>)"
+    "existing": "existing"
```

Comment:
Some languages will use different punctuation from `(` and `)` and/or won't include a space character. Instead of translating "existing" specifically, I think we should translate the entire string, something like:

```js
"{provided} (existing: {current})"
```

If it includes the entire string, I think we should also change the message key. What do you think about `entity.existingProperty` or `entity.duplicateProperty`? Or maybe even better, you could change `util.request.problem.409_17` from a single message to a group of messages that includes this one.

By the way, I think it makes sense not to include the bullet point in the string.